controller/rest/rest.go

```diff
@@ -1604,6 +1604,7 @@ func StartRESTServer(isNewCluster bool, isLead bool) {
 	r.GET("/v1/system/config", handlerSystemGetConfig)   // supported 'scope' query parameter values: ""(all, default)/"fed"/"local". no payload
 	r.GET("/v2/system/config", handlerSystemGetConfigV2) // supported 'scope' query parameter values: ""(all, default)/"fed"/"local". no payload. starting from 5.0, rest client should call this api.
 	r.GET("/v1/system/rbac", handlerSystemGetRBAC)       // skip API document
+	r.GET("/v1/system/alerts", handlerSystemGetAlerts)   // skip API document
```

Comment:
Since this is going to be a public API, you probably want to include API docs in this case.  
```suggestion
	r.GET("/v1/system/alerts", handlerSystemGetAlerts)
```

Reply:
change it to public interface and document it